Corona-Warn-App/src/main/res/layout/contact_diary_add_location_fragment.xml

```diff
@@ -4,7 +4,6 @@
     android:layout_height="match_parent"
     android:background="@color/colorBackground"
     android:fillViewport="true"
-    android:fitsSystemWindows="true"
```

Comment:
This was added by Ralph with ticket 5869. However, it broke the screen on Nokia device., I I removed it. Needs to be tested on Samsung.

Reply:
Checked that on Samsung S9 and looks fine, I find two other screens where is flag is used also 
1- fragment_edit_check_in
2- fragment_confirm_check_in
and in MainActivity layout and this is intended and should actually apply for all screens hosted by it 

Are these two screens working fine on your device?

Reply:
It was related to the keyboard, which does not come up on the two screen you mentioned. Thanks for checking though.